### PR TITLE
Add OVH as Diamond Sponsor #162

### PR DESCRIPTION
**Pull request by Nareshchoudhary02**
Add OVH as Diamond Sponsor
#162 done. Just add ovhcloud_logo.svg and that's it.😄

### CODE REVIEW

File: package-lock.json [hunk 1]
```diff
@@ -1,31 +1,61 @@
 {
   "name": "osday",
   "version": "0.1.0",
-  "lockfileVersion": 1,
+  "lockfileVersion": 3,
```
Comment (nicpuppa) [inline package-lock.json:4]: @Nareshchoudhary02 can you please revert these changes ?

**Review by nicpuppa — CHANGES_REQUESTED** [review]

**Review by nicpuppa — CHANGES_REQUESTED** [review]
@Nareshchoudhary02 can you please revert the changes on package-lock.json ?

**Comment by Nareshchoudhary02** [comment]
ok let me revert package-lock.json

**Review by nicpuppa — CHANGES_REQUESTED** [review]
@Nareshchoudhary02 also you forget to upload the logo into public folder

**Comment by Nareshchoudhary02** [comment]
Hello, I am new to the open source contribution community and I would greatly appreciate any guidance or advice on how to get started. Thank you

**Comment by nicpuppa** [comment]
> Hello, I am new to the open source contribution community and I would greatly appreciate any guidance or advice on how to get started. Thank you

What you have done so far is great! Looking for some good first issue, ask to the maintainers of the project to take an issue, fork the repo and open your first PR, these are the step to follow to contribute to an open-source project!

We also have a discord server (https://discord.com/invite/RTXr8A3eFn), feel free to join and ask whatever you want.

To conclude this issue, you need to upload the image file in the correct folder!

**Comment by Nareshchoudhary02** [comment]
> 

I uploaded a logo to the public folder but facing an issue because I requested too many merge requests it is messed up.

**Comment by nicpuppa** [comment]
@Nareshchoudhary02 can you please update the logo that you upload yesterday with this one ? 

OVHcloud_stacked_logo_white_RGB

**Comment by Nareshchoudhary02** [comment]
> @Nareshchoudhary02 can you please update the logo that you upload yesterday with this one ?
> 
> OVHcloud_stacked_logo_white_RGB

ok on it.

**Comment by Nareshchoudhary02** [comment]
> @Nareshchoudhary02 can you please update the logo that you upload yesterday with this one ?
> 
> OVHcloud_stacked_logo_white_RGB

It's done 😄. I have question how you converted got svg of image I want to know , because svg are more scalable.

**Comment by nicpuppa** [comment]
> It's done smile. I have question how you converted got svg of image I want to know , because svg are more scalable.

The company provide us the correct image (svg in this case)

**Pull request merged**